### PR TITLE
Added dozen of kernel nids and rearranged database

### PR DESCRIPTION
**Pull request by TheOfficialFloW**

### CODE REVIEW

**Comment by devnoname120** [comment]
If possible, please split in multiple commits the new NIDs and the rearrangement.

**Comment by TheOfficialFloW** [comment]
Ough, that's going to be hard and I'm also not up to doing that tbh.

File: db.yml [hunk 1]
```diff
@@ -1697,7 +1715,6 @@ modules:
           memset: 0xAB9BF5C
           rshift: 0x1D89F6C0
           snprintf: 0xAE7A8981
-          sprintf: 0x3DDBE2E1
```
Comment (devnoname120) [inline db.yml]: Why?

Comment (TheOfficialFloW) [inline db.yml]: This NID was added by me, but I have finally realized that it is not sprintf.

**Comment by devnoname120** [comment]
Human-readable diff: https://www.diffchecker.com/oJUF5DWC

**Comment by devnoname120** [comment]
See https://github.com/vitasdk/vita-headers/pull/138.

**Pull request closed without merging**